### PR TITLE
Support for ProMED posts on the Curator.

### PR DESCRIPTION
**Pull request by gbersani**
https://www.pivotaltracker.com/story/show/129926009
https://www.pivotaltracker.com/story/show/131455535

- Changes the collection used by the Curator to `posts`, which is generated by the scraper.
- Adds filters to ignore malformed post data.

Note: The `posts` collection still needs to be restored with the data parsed by the script on the deployed server. Related story: https://www.pivotaltracker.com/story/show/131455091

Perform a simple mongorestore with an existing `posts` collection on local to test the changes.

Update:
- Fetches data remotely from SPA.
- Removed article metadata on request.
- Modularized new collections.
- Adds foldable "content" data to the details view.

### CODE REVIEW

File: collections/posts.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+@PromedPosts = new Meteor.Collection "posts"
+
+if Meteor.isServer
+  ReactiveTable.publish "promedPosts", PromedPosts, {}
+
+  Meteor.publish "promedPosts", (limit, range) ->
+    query = {sourceDate: {$exists: true}}
```
Comment (gbersani) [inline collections/posts.coffee]: Updated in the latest commit.
It also gets normalized into addedDate and PublishDate to match closer to the "articles" schema, as well as leave this open for new sources in the future.

File: client/controllers/curatorInbox.coffee [hunk 2]
```diff
@@ -114,21 +133,28 @@ Template.curatorInboxSection.onCreated ->
         return ''
     },
     {
-      key: 'url'
-      description: 'The article\'s title.'
+      key: 'source'
+      description: 'The post\'s source metadata.'
       label: 'Title'
```
Comment (gbersani) [inline client/controllers/curatorInbox.coffee]: Updated.

**Comment by gbersani** [comment]
Updated the PR to modularize the collections and use the existing SPA mongodb as the source database for promed posts.

Post data is normalized and cached for performance and easy management.

The server needs to have it's Environment variable SPA_MONGO_URL set to allow EIDR-C to connect to SPA.

**Comment by nathanathan** [comment]
👍 Thanks Sal!

**Pull request merged**